repository_service_tuf_worker/repository.py

```diff
@@ -553,8 +553,11 @@ def add_targets(
 
             bin_targets[bins_name].append(db_target)
 
-        subtask = self._send_publish_targets_task(task_id)
-        self._update_task(bin_targets, subtask, update_state)
+        # If publish_targets doesn't exists it will be True by default.
+        publish_targets = payload.get("publish_targets", True)
+        if publish_targets is True:
+            subtask = self._send_publish_targets_task(task_id)
+            self._update_task(bin_targets, subtask, update_state)
```

Comment:
The `self._update_task` is the job that keeps updating the main task progress (`update_state`, not only the `sub_task` progress.
This will require some changes:
1. The `self._update_task` needs to be moved outside of the if the condition
2. In the `self._update_task` the parameter `subtask` needs to be `Optional[AsyncResult] = None`